### PR TITLE
Issue#481 Remove 'experimental' from SSR page

### PR DESCRIPTION
**Pull request by SaumyaBhushan**

### CODE REVIEW

File: src/pages/en/guides/server-side-rendering.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-**Server-side Rendering**, aka SSR, is enabled in Astro behind an experimental flag. When you enable SSR you can:
+**Server-side Rendering**, aka SSR, is enabled in Astro . When you enable SSR you can:
 
 - Implement sessions for login state in your app.
 - Render data from an API called dynamically with `fetch`.
 - Deploy your site to a host using an *adapter*.
 
-> SSR is marked as __experimental__ in Astro and changes will occur before it becomes stable. Use only if you can handle API changes.
+> SSR is new in Astro and changes will occur, before v1.0 stable release. Please keep up to date with API changes here.
```
Comment (sarah11918) [inline src/pages/en/guides/server-side-rendering.md:13]:
```suggestion
> SSR is new in Astro and changes will occur before v1.0 stable release. Please keep up to date with API changes here.
```
I just removed a comma here.

File: src/pages/en/guides/server-side-rendering.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: ~/layouts/MainLayout.astro
-title: Server-side Rendering (experimental)
+title: Server-side Rendering 
 i18nReady: true
 ---
 
-**Server-side Rendering**, aka SSR, is enabled in Astro behind an experimental flag. When you enable SSR you can:
+**Server-side Rendering**, aka SSR, is enabled in Astro . When you enable SSR you can:
```
Comment (sarah11918) [inline src/pages/en/guides/server-side-rendering.md:7]:
```suggestion
**Server-side Rendering**, aka SSR, is enabled in Astro. When you enable SSR you can:
```
Deleted space.

**Review by sarah11918 — COMMENTED** [review]
Thank you for helping! There are 2 quick fixes! :)

**Comment by SaumyaBhushan** [comment]
@sarah11918 applied quick fixes.

**Comment by sarah11918** [comment]
Thank you very much, and welcome to Team Docs! 🥳

**Pull request merged**